frontend/app/module/input/PollingStation/PollingStationLayout.test.tsx

```diff
@@ -41,7 +41,7 @@ describe("PollingStationLayout", () => {
         </ElectionProvider>
       </ElectionListProvider>,
     );
-    expect(await screen.findByText(pollingStationMock.name));
+    expect(await screen.findAllByText(pollingStationMock.name));
```

Comment:
`findAllByText()` is too broad. We should use [`within()`](https://testing-library.com/docs/dom-testing-library/api-within/) to do separate asserts for the navbar and the header.

When it's a link, we should use `getByRole('link', { name: '<text>' })`. The more specific we can make the locators (without getting into testing-the-implementation territory), the better our test coverage.

For the header we should switch to `screen.getByRole("heading", { level: 1, name: '<text>' })`.

The same thing needs to happen in `InputHomePage.test.tsx`.

Reply:
Improved in ac58fd5.